addons/behaviour/initSettings.sqf

```diff
@@ -2,8 +2,28 @@
     QGVAR(reactDeadBodies),
     "CHECKBOX",
     "React to dead bodies",
-    "Magnetar AI",
+    "AAIS",
     false,
     true,
     {}
 ] call CBA_Settings_fnc_init;
+
+[
+    QGVAR(useSmoke),
+    "CHECKBOX",
+    "Use smoke grenades?",
+    "AAIS",
+    false,
+    true,
+    {}
+] call CBA_Settings_fnc_init;
+
+[
+    QGVAR(grenadeChance),
+    "SLIDER",
+    "Grenade chance in %",
+    "AAIS",
+    [0, 100, 45, 0],
+    true,
+    {}
+] call CBA_fnc_addSetting;
```

Comment:
All these settings can be in nice 3den dialogs instead of CBA settings. CBA settings only have a few types which don't work for all the settings we want to do, and keeping them in one place would be helpful for mission makers

Reply:
I agree. For me 3DEN attributes are the last step. CBA settings are good in order to have a system ready in place and once it is finished, move it coherently to 3DEN.

Reply:
Perfectly fine with that